components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -122,34 +172,49 @@ void Nudging::time_interpolation (const int time_s) {
   auto T_mid_aft = view_2d<Real>("",NudgingData_aft.T_mid.extent_int(0),
```

Comment:
These views are created on device at every timestep. That's not great, especially on GPUs. You should allocate them once and for all at init time, and reuse them over and over.

Reply:
To be fair though, I am not sure why you need these local deep copies. Can't you just use, say, `NudgingData_aft.T_mid` everywhere you use `T_mid_aft` (and similarly for other vars)?